src/plugins/safeSnap/index.ts

```diff
@@ -130,9 +137,106 @@ export default class Plugin {
     };
   }
 
-  async getModuleDetails(network: string, moduleAddress: string) {
+  async getModuleDetailsReality(network: string, moduleAddress: string) {
     const provider: StaticJsonRpcProvider = getProvider(network);
-    return getModuleDetails(provider, network, moduleAddress);
+    return getModuleDetailsReality(provider, network, moduleAddress);
+  }
+
+  async validateUmaModule(network: string, umaAddress: string) {
+    if (!isAddress(umaAddress)) return 'reality';
+    
+    const provider: StaticJsonRpcProvider = getProvider(network);
+    const moduleContract = new Contract(
+      moduleAddress,
```

Comment:
The var `moduleAddress` does not exists here, I assume you mean `umaAddress`
```suggestion
      umaAddress,
```